src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -14,6 +14,29 @@
 
     public void moveRobot(Robot robot, int toX, int toY) {
         //your code
+        if (toX - robot.getX() > 0) {
+            while (robot.getDirection() != Direction.RIGHT) {
+                robot.turnRight();
+            }
+        } else if (toX - robot.getX() < 0) {
```

Comment:
you can just use "else" here

Reply:
@RomanDubovskyi but this means it would turn him even the differences between toX and robot.getX() = 0. whouldn't that be unnecessary due to fact that he won't need to move in the direction he would turn in this case?

Reply:
done

Reply:
yep, you're right, in this case we can afford it, cause performance impact won't be significant, but code will be shorter

Reply:
@RomanDubovskyi thanks for your attention!